tests/visual/scenes/graphics/graphics-pixel-line.scene.ts

```diff
@@ -0,0 +1,24 @@
+import { Graphics } from '../../../../src/scene/graphics/shared/Graphics';
+
+import type { Renderer } from '../../../../src/rendering/renderers/types';
+import type { Container } from '../../../../src/scene/container/Container';
+import type { TestScene } from '../../types';
+
+export const scene: TestScene = {
+    it: 'should render a pixel line correctly',
+    create: async (scene: Container, _renderer: Renderer) =>
+    {
+        const graphics = new Graphics();
+
+        graphics
```

Comment:
Hmm this doesn't match the snapshot image. I only see the first black line. Am I missing something?